src/main/java/mate/academy/dao/UserDao.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.User;
+
+public interface UserDao {
+    User add(User user);
+
+    Optional<User> get(Long id);
```

Comment:
This method and the getAll() method are not used anywhere. Remove it from here and from services, too. Write only the code you need to solve your problem

Reply:
Done